docs/_plugins/search_index.rb

```diff
@@ -79,7 +79,10 @@ def editions_changed?(local_editions)
   if res.is_a?(Net::HTTPSuccess)
     data = JSON.parse(res.body)
     remote_editions = data['meta']['aggregations']['editions']
-    return !local_editions.to_set.subset?(remote_editions.to_set)
+    outdated_editions = ['Spark NLP 2.1', 'Healthcare NLP 2.0']
```

Comment:
I see the same `outdated_editions` in another method. Maybe it makes sense to extract it to a constant?

Reply:
Used a constant variables for both